sql/migrations/20210515110157_world.sql

```diff
@@ -0,0 +1,851 @@
+
+-- Explosive Sheep 2675 (credit cmangos)
+INSERT INTO `creature_ai_events` (`id`, `creature_id`, `condition_id`, `event_type`, `event_inverse_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action1_script`, `action2_script`, `action3_script`, `comment`) VALUES (267501, 2675, 0, 11, 0, 100, 0, 0, 0, 0, 0, 267501, 0, 0, 'Explosive Sheep - Cast Explosive Sheep Passive and Stealth Detection on Spawn');
+INSERT INTO `creature_ai_scripts` (`id`, `delay`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) VALUES (267501, 0, 15, 4051, 0, 0, 0, 0, 0, 0, 4, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Explosive Sheep - Cast Spell');
+INSERT INTO `creature_ai_scripts` (`id`, `delay`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) VALUES (267501, 0, 15, 8279, 0, 0, 0, 0, 0, 0, 4, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Explosive Sheep - Cast Spell');
+
+-- Naga Explorer 1907 (credit cmangos)
+INSERT INTO `creature` (`guid`, `id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecsmin`, `spawntimesecsmax`, `wander_distance`, `health_percent`, `movement_type`, `patch_max`) VALUES
+(2090, 1907, -13669.1, 616.477, 7.57179, 3.58152, 300, 300, 5, 100, 1, 10),
+(2093, 1907, -13701, 633.343, 7.3798, 3.06592, 300, 300, 5, 100, 1, 10),
+(2554, 1907, -13705.4, 574.627, 18.7638, 4.46864, 300, 300, 5, 100, 1, 10),
+(180, 1907, -13783.5, 645.277, 14.8763, 5.95696, 300, 300, 5, 100, 1, 10);
+UPDATE `creature` SET `position_x` = -13780.404297, `position_y` = 498.528076, `position_z` = 69.540359, `wander_distance` = 5, `movement_type` = 1 WHERE `guid` = 1966;
+
+-- Tethis 730 (credit cmangos)
+INSERT INTO `creature` (`guid`, `id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecsmin`, `spawntimesecsmax`, `wander_distance`, `health_percent`, `movement_type`, `patch_max`) VALUES
+(2588, 730, -12978.6, 411.841, 21.7456, 3.43503, 420, 600, 15, 100, 1, 10),
+(565, 730, -13017.2, 193.101, 22.7814, 1.45268, 420, 600, 15, 100, 1, 10);
+UPDATE `creature` SET `wander_distance` = 20, `movement_type` = 1 WHERE `id` = 730;
+DELETE FROM `pool_creature_template` WHERE `id` = 730;
+INSERT INTO `pool_creature_template` (`id`, `pool_entry`, `chance`, `description`, `patch_max`) VALUES
+(730, 125, 0, 'Tethis (730)', 10);
+DELETE FROM `pool_template` WHERE `entry` = 1266;
+INSERT INTO `pool_template` (`entry`, `max_limit`, `description`) VALUES
+(125, 1, 'Tethis (730)');
+
+-- Southsea Cannoneer Should Not Drop Lvl 20 Items
+UPDATE `creature_loot_template` SET `item` = 30041 WHERE `entry` = 3382 AND `item` = 30040 AND `groupid` = 0;

```

Comment:
The `item` field is just a placeholder when its a reference, and not an actual item. You want to update `mincountOrRef` in addition (it should be equal to the `reference_loot_template` entry you want to include, but with a minus in front).

Reply:
Placeholder 30040 is for lvl 20 items, 30041 is for the correct lvl range items.

Reply:
I get that but you are only changing the `item` field which does not do anything when its a reference. You need to also set `mincountOrRef` to -30041.